server/src/routes/tableaux/european-projects/index.js

```diff
@@ -533,13 +533,15 @@ router.route('/european-projects/analysis-positioning-top-10-beneficiaries')
     const total_fund_eur = data.reduce((acc, el) => acc + el.total_fund_eur, 0);
     const dataReturn = {
       total_fund_eur,
-      top10: data.slice(0, 10).map((el) => {
-        acc += el.total_fund_eur;
-        return {
-          ...el,
-          influence: acc / total_fund_eur * 100,
+      top10: data.map((el) => {
+        if (el.id !== 'ZOE' && el.id !== 'ZOI') {
+          acc += el.total_fund_eur;
+          return {
+            ...el,
+            influence: acc / total_fund_eur * 100,
+          }
         }
-      })
+      }).filter((el) => el).slice(0, 10)
```

Comment:
.filter((el) => el) is a way to filter non null element ?

Reply:
yes